inc/redis_alloptions/namespace.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace HM\Platform\Redis_Alloptions;
+
+function bootstrap() {
+	add_action( 'added_option', __NAMESPACE__ . '\maybe_clear_alloptions_cache' );
```

Comment:
Elsewhere we usually escape `\` as `\\`, i.e. ` __NAMESPACE__ . '\\maybe_clear_alloptions_cache`